server.js

```diff
@@ -0,0 +1,20 @@
+const express = require("express");
+
+const fs = require("fs");
+
+const app = express();
+
+app.get('/files/:filename', (req, res) => {
+  
+    const filePath = `./files/${req.params.filename}`;
+
+  const fileStream = fs.createReadStream(filePath);
```

Comment:
Where is your error handling? What happens if the file the user requested does not exists?